matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/send/MarkdownParser.kt

```diff
@@ -27,7 +28,8 @@ import javax.inject.Inject
  * If any change is required, please add a test covering the problem and make sure all the tests are still passing.
  */
 internal class MarkdownParser @Inject constructor(
-        private val parser: Parser,
+        private val advancedParser: Parser,
+        @SimpleCommonmarkParser private val simpleParser: Parser,
```

Comment:
For parity do you think it's worth creating a AdvancedCommonmarkParser annotation class?